aws/logs_monitoring/lambda_function.py

```diff
@@ -588,6 +616,10 @@ def split(events):
             trace_payloads.append(trace_payload)
         else:
             logs.append(event)
+
+    """Dedup telemetry tags"""
+    DD_FORWARDER_TELEMETRY_TAGS = list(set(DD_FORWARDER_TELEMETRY_TAGS))
```

Comment:
It's better to avoid the duplicates in the first place. I think you can make `DD_FORWARDER_TELEMETRY_TAGS` to be a dict instead of list, such as:

```python
{
    "forwardername": ...,
    "forwarder_memorysize": ....,
    "forwarder_version": ...,
    "source": ...
}
```

BTW, these four tags are the only tags I expect we add to the telemetries, to be conservative on cardinality.